TelepresenceBot/movement-unit/movement-unit.ino

```diff
@@ -0,0 +1,142 @@
+#include <AFMotor.h> // from https://github.com/adafruit/Adafruit-Motor-Shield-library
+#include <Servo.h> 
+
+AF_DCMotor motorRight1(3);
+AF_DCMotor motorRight2(2);
+AF_DCMotor motorLeft1(4);
+AF_DCMotor motorLeft2(1);
+
+const char COMMAND_FORWARD = 'w';
+const char COMMAND_BACKWARD = 's';
+const char COMMAND_TEST = 't';
+const int MAX_SPEED = 255;
+const int DELTA_SPEED = 50;
+
+int currentDirection = RELEASE;
```

Comment:
`RELEASE` is a constant for the `AF_DCMotor` direction which means "stop".
From the official documentation:
```
RELEASE - Stop the motor.  This removes power from the motor and is equivalent to setSpeed(0).  The motor shield does not implement dynamic breaking, so the motor may take some time to spin down
```
(https://learn.adafruit.com/adafruit-motor-shield/af-dcmotor-class)